cli/src/main.rs

```diff
@@ -108,13 +108,20 @@ async fn main() -> Result<()> {
             for (cluster_name, cluster) in &state.clusters {
                 for (backend_id, backend) in &cluster.backends {
                     if let Some((timestamp, state)) = backend.state_timestamp() {
-                        if state != BackendState::Swept {
+                        if state == BackendState::Swept || state == BackendState::Terminated {
```

Comment:
Should we include all terminal states here, not just swept and terminated? (i.e. `Exited`, `Failed`, `ErrorLoading`, `ErrorStarting`, `TimedOutBeforeReady`, and `Lost`)